src/test/java/apicurito/tests/configuration/templates/ApicuritoInstall.java

```diff
@@ -23,7 +23,7 @@ public static void cleanNamespace() {
         TestConfiguration.printDivider("Deleting namespace resources");
 
         try {
-            OpenShiftUtils.getInstance().customResourceDefinitions().withName("apicuritoes.apicur.io").delete();
```

Comment:
why this change? The old method is not working well?

Reply:
The old method didn't delete the CRD